app/src/main/java/comp3350/mealbuddy/objects/Exercise.java

```diff
@@ -6,9 +6,10 @@
         Low, Medium, High,
     }
 
+    public String name;
```

Comment:
Do we want to just leave this at name or have a set of enums like run, weights...etc.   I think I'm a fan of just having a name and keeping it pre minimal as it isn't the focus of the app.  later in another iteration we can do more interesting stuff but prob not since again its not the main feature.

Reply:
TBH, I didn't want to think about Exercises. Just wanted to tack on enough info to make the stub database in the future. We can make it a set list of exercises in the future

Reply:
I am 100% ok with this sentiment haha